typespec/package.json

```diff
@@ -7,7 +7,7 @@
     "@azure-tools/typespec-autorest": "~0.46.0",
     "@azure-tools/typespec-azure-core": "~0.46.0",
     "@azure-tools/typespec-azure-resource-manager": "~0.46.0",
-    "@azure-tools/typespec-providerhub": "^0.46.0",
+    "@azure-tools/typespec-providerhub": "^0.47.0",
```

Comment:
I think I messed up this wildcard and it was missed in code review. This should look like the others. 

https://github.com/radius-project/radius/commit/3492b53d0270fe946cded23e8805d915bcb1c29a

😢 😢 😢